community/cypher/cypher/src/test/scala/org/neo4j/cypher/GraphDatabaseTestSupport.scala

```diff
@@ -73,6 +76,13 @@ trait GraphDatabaseTestSupport extends CypherTestSupport with GraphIcing {
     graph.inTx(r.getGraphDatabase.index.forRelationships(idxName).add(r, key, value))
   }
 
+  def createIndex(labelName: String, propertyKeyName: String) {
```

Comment:
There already is a `createIndex` defined implicitly on`graph`. So you can do `graph.createIndex("foo", "bar")`